roles/foreman_provisioning/tasks/main.yml

```diff
@@ -145,10 +144,28 @@
   shell: >
     {{ foreman_provisioning_hammer }} settings set --name query_local_nameservers --value true
 
-# testing both --name and --title for backward compatibility
+# make installation media available
+- name: 'assign organization and location to all seeded media'
```

Comment:
I think this is a usability issue and should be fixed in out seeds.

Reply:
I tend to agree, I opened a ticket and will add a comment to the code, this shouldn't prevent having this workaround meanwhile...